### DISCUSSION TITLE
The Quadrature of the sampler is a sampling method

### DISCUSSION THREAD

**Discussion by Ating24** [idaholab/moose#27042] [Q&A Tools]
What is the method for selecting sampling points in Quadrature? Does Quadrature mean orthogonality or integration?

**Comment by GiudGiud** [comment]
Hello

It depends on the context. 

There s sampling in the stochastic tools for various purposes. 

There s a quadrature for numerical integration in the finite element equations solve. 
In that context we sample every point in the quadrature chosen at the order defined. 


I imagine you're considering a quadrature in the stochastic tools context?

Guillaume

**Reply by Ating24** [reply]
Yes, when I was using polynomial chaos for uncertainty quantification, the sampler chose Quadrature

**Reply by grmnptr** [reply]
https://mooseframework.inl.gov/source/samplers/QuadratureSampler.html
